src/hello.erl

```diff
@@ -51,7 +52,11 @@ start() ->
 % @doc Callback for application behaviour.
 start(_Type, _StartArgs) ->
     {ok, Supervisor} = hello_supervisor:start_link(),
-    hello_metrics:start_subscriptions(),
+    case application:get_env(hello, metrics) of
```

Comment:
I think, there should be rational default preconfigured in .app.src file.

Other reason, is, that it should be seen all possible configuration in .app.src file without to search all in code.

Other small change, it should be possible to disable without unset enviroment, for example with {metrics, []}.

Than, undefined clause is not needed?